drivers/media/i2c/imx500.c

```diff
@@ -1903,13 +1934,190 @@ static void imx500_clear_fw_network(struct imx500 *imx500)
 	v4l2_ctrl_activate(imx500->device_id, false);
 }
 
+static int __must_check imx500_spi_write(struct imx500 *state, const u8 *data,
+					 size_t size)
+{
+	if (size % 4 || size > ONE_MIB)
```

Comment:
nit: `size % 4` would be the same as `size & 3`. The compiler may well do that anyway.